deps/wazuh_testing/wazuh_testing/tools/system.py

```diff
@@ -480,31 +479,49 @@ def install_package(self, host, url, system='ubuntu'):
 
-        if system == 'windows':
-            if url.lower().endswith(extension):
-                result = self.get_host(host).ansible("win_package", f"path={url} arguments=/passive", check=False)
+        if use_npm:
+            if system == 'macos':
+                cmd = f"PATH=/opt/homebrew/bin:$PATH npm install -g {url}"
+                result = self.get_host(host).ansible("shell", cmd, check=False)
+            elif system == 'windows':
+                cmd = f"npm install -g {url}"
+                result = self.get_host(host).ansible("win_shell", cmd, check=False)
             else:
-                result = self.get_host(host).ansible("win_package", f"path={url} arguments=/S", check=False)
-        elif system == 'ubuntu':
-            result = self.get_host(host).ansible("apt", f"deb={url}", check=False)
-            if result['changed'] and result['stderr'] == '':
-                result = True
-        elif system == 'centos':
-            result = self.get_host(host).ansible("yum", f"name={url} state=present "
+                cmd = f"npm install -g {url}"
+                result = self.get_host(host).ansible("shell", cmd, check=False)
+
+            logging.info(f"npm package installed result {result}")
+            return result
+
+        else:
+            result = False
+            extension = '.msi'
+
+            if system == 'windows':
+                if url.lower().endswith(extension):
+                    result = self.get_host(host).ansible("win_package", f"path={url} arguments=/passive", check=False)
+                else:
+                    result = self.get_host(host).ansible("win_package", f"path={url} arguments=/S", check=False)
+            elif system == 'ubuntu':
+                result = self.get_host(host).ansible("apt", f"deb={url}", check=False)
+                if result['changed'] and result['stderr'] == '':
+                    result = True
+            elif system == 'centos':
+                result = self.get_host(host).ansible("yum", f"name={url} state=present "
                                                  'sslverify=false disable_gpg_check=True', check=False)
-        elif system == 'macos':
-            package_name = url.split('/')[-1]
-            result = self.get_host(host).ansible("command", f"curl -LO {url}", check=False)
-            cmd = f"installer -pkg {package_name} -target /"
-            result = self.get_host(host).ansible("command", cmd, check=False)
+            elif system == 'macos':
+                package_name = url.split('/')[-1]
+                result = self.get_host(host).ansible("command", f"curl -LO {url}", check=False)
+                cmd = f"installer -pkg {package_name} -target /"
+                result = self.get_host(host).ansible("command", cmd, check=False)
 
-        logging.info(f"Package installed result {result}")
+            logging.info(f"Package installed result {result}")
 
-        return result
+            return result
```

Comment:
You should remove the return on line 498 (inside the if) and tab to the left the return on line 524 (inside the else) in order to have a single return in the function.